home/tests/test_content_import_export.py

```diff
@@ -847,6 +847,42 @@ def test_cpi_validation_failure_update(self, csv_impexp: ImportExport) -> None:
             == "Validation error: {'translation_key': ['“BADUUID” is not a valid UUID.']}"
         )
 
+    def test_ContentPageIndex_required_fields(self, csv_impexp: ImportExport) -> None:
+        """
+        Importing an CSV file with only the required fields shoud not break
+
+        """
+
+        csv_bytes = csv_impexp.import_file("contentpage_index_required_fields.csv")
+        content = csv_impexp.export_content()
+        src, dst = csv_impexp.csvs2dicts(csv_bytes, content)
+
+        content_pages = ContentPageIndex.objects.all()
```

Comment:
Since we know what the source data is, we can be a lot more explicit with the assertions:

```python
[main_menu] = ContentPageIndex.objects.all()
assert main_menu.slug == "main-menu"
```

The first line will fail if we don't have exactly one `ContentPageIndex`, and the second will fail if the slug doesn't match.

Reply:
I kept this line since I use that to check how many pages/ indexes we have, but I could find a work around to get that data directly from `csv_bytes`, not sure.

Reply:
Comparing the original and re-exported CSVs will ensure that the line/page counts are the same, and the `[main_menu] = ...` assignment will raise an exception if the sequence on the right doesn't have exactly one element. 🙂